tests/suite/system_suite_test.go

```diff
@@ -173,6 +175,12 @@ func setup(cfg setupConfig, extraInstallArgs ...string) {
 		}
 		installCfg.ImageTag = *imageTag
 		installCfg.ImagePullPolicy = *imagePullPolicy
+		// if we are installing from a local chart, we can't specify a chart version
+		installCfg.ChartVersion = ""
```

Comment:
did you mean like this @sjberman ?

Reply:
Yeah, or don't specify it at all since the default value of `installCfg.ChartVersion` is an empty string.